### DISCUSSION TITLE
Weird info output when tests are finished

### DISCUSSION THREAD

**Discussion by sylouuu** [pestphp/pest#1169] [Q&A] [answered]
Hi everyone,

Just asking, do you have an idea why this `info` is there? I've searched everywhere, the codebase, even in `vendor`, Google...

![CleanShot 2024-06-08 at 18 58 26@2x](https://github.com/pestphp/pest/assets/2300259/a8800eee-095d-4137-b48e-72f4b1b79348)

Bests

**Comment by potsky** [comment]
I have the same output and no idea!

**Comment by faissaloux** [comment]
Can you tell us how to reproduce this, and which version you using?

**Reply by sylouuu** [reply]
> Can you tell us to reproduce this, and which version you using?

```
"pestphp/pest": "^2.28",
```

Found in `src/KernelDump.php`
![CleanShot 2024-06-09 at 07 59 24@2x](https://github.com/pestphp/pest/assets/2300259/b4591bcf-a1ac-454b-b959-2eda12da1afa)
![CleanShot 2024-06-09 at 08 03 30@2x](https://github.com/pestphp/pest/assets/2300259/e73c9a30-9274-4073-95de-4f25c393c5cd)

**Reply by faissaloux** [reply]
I need the current version installed on your project. Can you show me the output of `composer show pestphp/pest`.

**Reply by faissaloux** [reply]
Anyway this doesn't appear on the latest version `v2.34.7`, this must have been fixed.

**Comment by potsky** [comment]
We have the same `INFO f.` at start

![Screenshot 2024-06-09 - 17-59-03@2x](https://github.com/pestphp/pest/assets/408237/020f835e-68ab-4ae4-a3f6-1de02532b9e7)

**Comment by faissaloux** [comment]
I couldn't reproduce this on my machine, maybe you need a fresh install.

**Comment by potsky** [comment]
@faissaloux we will find where It comes from ;-)

**Reply by potsky** [reply, marked as answer]
Found!
Just a mistake at the beginning of a PHP file... `f<?php` instead of `<?php`

**Reply by nunomaduro** [reply]
lol

**Reply by potsky** [reply]
yep I'm a little ashamed 🫣
Thanks for your amazing work @nunomaduro!
